server/__tests__/db.test.ts

```diff
@@ -0,0 +1,18 @@
+import request from 'supertest';
+import { createServer } from '../server';
+const mockingoose = require('mockingoose');
```

Comment:
this one cannot be changed, because `mockingoose` doesn't support esm https://stackoverflow.com/questions/70156753/typeerror-0-mockingoose-default-is-not-a-function-mockingooose